js/packages/openinference-instrumentation-openai/src/instrumentation.ts

```diff
@@ -334,17 +337,64 @@ function getLLMInputMessagesAttributes(
   body: ChatCompletionCreateParamsBase,
 ): Attributes {
   return body.messages.reduce((acc, message, index) => {
+    const messageAttributes = getChatCompletionInputMessageAttributes(message);
     const index_prefix = `${SemanticConventions.LLM_INPUT_MESSAGES}.${index}`;
```

Comment:
Did you want to add a dot here so it's consistent with your other changes below?
```suggestion
    const index_prefix = `${SemanticConventions.LLM_INPUT_MESSAGES}.${index}.`;
```

Reply:
Ah yes I did, thanks